security_monkey/datastore.py

```diff
@@ -288,6 +298,26 @@ class IgnoreListEntry(db.Model):
     tech_id = Column(Integer, ForeignKey("technology.id"), nullable=False, index=True)
 
 
+class ExceptionLogs(db.Model):
+    """
+    This table stores all exceptions that are encountered, and provides metadata and context
+    around the exceptions.
+    """
+    __tablename__ = "exceptions"
+    id = Column(BigInteger, primary_key=True)
+    source = Column(String(256), nullable=False, index=True)
+    occurred = Column(DateTime, default=datetime.datetime.utcnow(), nullable=False)
+    ttl = Column(DateTime, default=(datetime.datetime.utcnow() + datetime.timedelta(days=10)), nullable=False)
+    type = Column(String(256), nullable=False, index=True)
+    message = Column(String(512))
+    stacktrace = Column(Text)
+    region = Column(String(64), nullable=True, index=True)
```

Comment:
Item.region is only 32 chars. Odd to have the same data stored with different lengths.